website/docs/advanced/impression_data.md

```diff
@@ -0,0 +1,95 @@
+---
+id: impression_data
+title: Impression data
+---
+
+Impression data represents that unleash uses locally in order to resolve a feature toggle to true or false. This data does not leave your application, and the data is not sent to unleash - but it is used for the purposes of resolving feature toggles.
```

Comment:
I think you might have forgotten a word or two in the first sentence there 😅 What does impression data represent?

Reply:
Woops! Yes that is totally correct.